packages/encryption/src/lib/utils.ts

```diff
@@ -16,29 +18,39 @@ export function isValidBooleanExpression(
   | SolRpcConditions 
   | UnifiedAccessControlConditions
 ): boolean {
+  console.log('expression');
+  console.log(expression);
   const STATES = { START: "start", CONDITION: "condition", OPERATOR: "operator" };
 
   let currentState = STATES.START;
   for (const token of expression) {
+    console.log('token- ', token);
     switch (currentState) {
       case STATES.START:
       case STATES.OPERATOR:
+        console.log(currentState);
         if (isTokenOperator(token)) {
+          console.log('invalid- 1');
           return false;
         }
         // Nested conditions expression
         if (Array.isArray(token) && !isValidBooleanExpression(token)) {
+          console.log('invalid- 2');
           return false;
         }
         currentState = STATES.CONDITION;
         break;
       default:
+        console.log('STATES.CONDITION');
         if (!isTokenOperator(token)) {
+          console.log('invalid- 3');
           return false;
         }
         currentState = STATES.OPERATOR;
     }
   }
 
-  return currentState === STATES.CONDITION;
+  const res = currentState === STATES.CONDITION;
+  console.log('isValidBooleanExpression- ', res);
+  return res;
```

Comment:
Can you remove all the console logs?

Reply:
I think he removed them before you could request the changes :)